src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -48,13 +52,21 @@ public void observeDangerZone(MovedEvent event, EntityRef player) {
         }
     }
 
+    /**
+     * Modifies the walking/running speed of the player.
```

Comment:
Well, that `modifySpeed` modifies the speed should be no surprise. However, in what way does it modify it? In particular, we are **decreasing** the **walk speed**.